internal/network/manifests_generator.go

```diff
@@ -340,3 +348,30 @@ func (m *ManifestsGenerator) AddDisableVmwareTunnelOffloading(ctx context.Contex
 	}
 	return nil
 }
+
+const redirectTelemeterStageManifest = `
```

Comment:
Move to the top next to all other consts

Reply:
They aren't all grouped on top of the file, for example in `AddDisableVmwareTunnelOffloading` the coasts are defined just before the method.

I don't mind moving it to the top though.